src/python/foglamp/__main__.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import foglamp.core.core_daemon
```

Comment:
ah! I missed that. Yes this will shadow the python-daemon's daemon module.

Yeah, here we either have to keep it as `core_daemon` itself or just another name may be`daemonizer`